python_scripts/repeatable/strip_whitespace.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+
+# This script works through a csv of offending content and removes whitespace from the specified field(s).
+# For help on available arguments and options:
+# `python repeatable/strip_whitespace.py -h`.
+import argparse
+import os
+import sys
+
+from asnake.aspace import ASpace
+from dotenv import load_dotenv, find_dotenv
+from loguru import logger
+from pathlib import Path
+from python_scripts.utilities import read_csv, record_error
+
+# Logging
+logger.remove()
+log_name = __file__.rsplit('/',1)[1].replace('.py', '')+'_{time:YYYY-MM-DD}'
+log_path = Path(f'./logs/{log_name}.log')
+logger.add(str(log_path), format="{time}-{level}: {message}")
+
+# Find  and load environment-specific .env file
+env_file = find_dotenv(f'.env.{os.getenv("ENV", "dev")}')
+load_dotenv(env_file)
+
+# Define functions
+def archival_objects(repo): return repo.archival_objects
+def digital_objects(repo): return repo.digital_objects
+def resources(repo): return repo.resources
+
+# Create a dictionary to map arguments to functions
+func_dict = {
+    "archival_objects": archival_objects,
+    "digital_objects": digital_objects,
+    "resources": resources
+}
+
+def parseArguments():
```

Comment:
We've been adding this function to quite a few of our scripts. Do you think it'd be worth adding to our utilities.py file? With the option of adding custom arguments per script when needed? (keeping -dR, --version as default, only overriding if necessary)

Reply:
Probably worth filing a backlog ticket for, but I was already introducing a lot here that wasn't truly necessary for the functioning of this particular data cleanup task, so I didn't/don't want it to grow more.  But it is a good call.  I also had the same though about the type argument/functions I added here - I can see us wanting to repurpose them in the future.

Reply:
Created #74